index.html

```diff
@@ -36,7 +36,255 @@ <h2 class="sd">
+                    </div>
+                    <div class="card-lists">
+                        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+                        <div class="items">
+                        <ul>
+                            <li>Ruby on rails</li>
+                            <li>Css</li>
+                            <li>Java script</li>
+                            <li>Html</li>
+                        </ul>
+                    </div>
+                    <div class="button-cont">
+                        <button type="button" class="potfolio-but">See Project</button>
+                    </div>
+                    </div>
+                   </div>
+        </section>
+
+        <section>
+            <div class="container-card">
+            <div class="placeholder">         
+            </div>
+            <div class="card-lists">
+                <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+                <div class="items">
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>Css</li>
+                    <li>Java script</li>
+                    <li>Html</li>
+                </ul>
+            </div>
+            <div class="button-cont">
+                <button type="button" class="potfolio-but">See Project</button>
+            </div>
+
+        </div>
+    </div>
+</section>
+
+<section>         
+    <div class="container-card">
+    <div class="placeholder">         
+    </div>
+    <div class="card-lists">
+        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <div class="items">
+        <ul>
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Java script</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <button type="button" class="potfolio-but">See Project</button>
+    </div>
+
+    </div>
+</div>
+</section> 
+
+<section>            
+    <div class="container-card">
+    <div class="placeholder">         
+    </div>
+    <div class="card-lists">
+        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <div class="items">
+        <ul>
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Java script</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <button type="button" class="potfolio-but">See Project</button>
+    </div>
+
+    </div>
+</div>
+</section>
+
+<section>          
+    <div class="container-card">
+    <div class="placeholder">         
+    </div>
+    <div class="card-lists">
+        <div><h3>Multi-Post Stories Gain+Glory</h3></div>
+        <div class="items">
+        <ul>
+            <li>Ruby on rails</li>
+            <li>Css</li>
+            <li>Java script</li>
+            <li>Html</li>
+        </ul>
+    </div>
+    <div class="button-cont">
+        <button type="button" class="potfolio-but">See Project</button>
+    </div>
```

Comment:
- [ ] As required by the previous reviewer, Please make sure this button and every other button on the page has the styling for the state interactions (Hover, Pressed, disabled).

![image](https://user-images.githubusercontent.com/63972670/160213942-a8ad1082-cda4-426b-b4d9-70b2f8aceed9.JPG)